mappings/1.3.tiny

```diff
@@ -22418,8 +22418,8 @@ c	net/minecraft/C_65ohdyrnc	net/minecraft/client/render/model/entity/BoatModel
 c	net/minecraft/C_65ojwbeub	net/minecraft/client/render/HeldItemRenderer
 	f	Lnet/minecraft/client/Minecraft;	f_54vqpmuec	f_54vqpmuec
 	f	Lnet/minecraft/C_72owraavl;	f_86ygsojqo	item
-	f	F	f_79oewarrl	using
-	f	F	f_09utkrzgg	usingDelta
+	f	F	f_79oewarrl	holdAnimationTicks
```

Comment:
Are these fields used for animation while holding an item, or only when you're using the item? If it's the latter, "use"/"using" is more in line with our conventions than "hold".